halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/persistentStorage/S3PersistentStore.java

```diff
@@ -25,6 +25,7 @@
 @Data
 @EqualsAndHashCode(callSuper = false)
 public class S3PersistentStore extends PersistentStore {
+  private String accountName;
```

Comment:
I noticed this pattern somewhere else so I didnt say anything since I figured it was a standard pattern.

However I'd like to note that it could be that the account for the storage is not something that you want to deploy infrastructure to (other than perhaps Spinnaker). So you wouldnt necessarily want it visible in clouddriver.